src/app-gocardless/README.md

```diff
@@ -3,7 +3,7 @@
 Find in [doc](https://docs.google.com/spreadsheets/d/1ogpzydzotOltbssrc3IQ8rhBLlIZbQgm5QCiiNJrkyA/edit#gid=489769432) what is id of bank which you want to integrate
 
 Add the `institution_id` and your name to list of possible options in the frontend
-project `actual/packages/loot-design/src/components/modals/NordigenExternalMsg.js`
+project `actual/packages/loot-design/src/components/modals/GoCardlessExternalMsg.js`
```

Comment:
It’s not `loot-design` anymore

Reply:
Good catch. https://github.com/actualbudget/actual-server/pull/231/commits/39522714a741bf34e6907c823e04b25200421b8f